<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/7.png);</string>
   </property>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>288</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordlineEdit">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>236</y>
      <width>137</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back1.png);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelpushButton">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>516</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitpushButton">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>516</y>
      <width>49</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>288</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usrnamelineEdit">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>192</y>
      <width>137</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back1.png);</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>196</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);
background-image: url(:/new/prefix1/back1.png);</string>
    </property>
    <property name="text">
     <string>注册新账号</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="findpwdButton">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>236</y>
      <width>53</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);
background-image: url(:/new/prefix1/back1.png);</string>
    </property>
    <property name="text">
     <string>找回密码</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="iplineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>432</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="portlineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>108</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/back1.png);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
